galaxyui/src/app/content-detail/content-header/content-header.component.ts

```diff
@@ -137,7 +137,8 @@ export class ContentHeaderComponent implements OnInit {
             iconClass: 'pficon-repository',
             tooltip: 'Collection',
             name: this.collection.name,
-            description: this.collection.latest_version.metadata.description,
+            description:
+                this.collection.latest_version.metadata.description || '',
```

Comment:
Is this the right fix, or should it have been an import error for the metadata to exclude a description in the first place? I did not think it was supposed to be optional.

Reply:
I looked into this and the description isn't required in the spec, so the importer is currently doing the right thing by ignoring it.